docs/examples/tutorial/string/return_memview.py

```diff
@@ -0,0 +1,9 @@
+def process_byte_data(data: cython.uchar[:]):
+    # ... process the data, here, dummy processing.
+    return_all: cython.bint = (data[0] == 108)
```

Comment:
I don't think it helps readers to have this variable typed.
```suggestion
    return_all = (data[0] == 108)
```

Reply:
cython will infer the type of the variable to object not boolean, so the produced c code will not be identical

```
static PyObject *__pyx_pf_{num_fname}_process_byte_data(CYTHON_UNUSED PyObject *__pyx_self, __Pyx_memviewslice __pyx_v_data) {
-   PyObject *__pyx_v_return_all = NULL;
+   int __pyx_v_return_all;
    PyObject *__pyx_r = NULL;
    __Pyx_RefNannyDeclarations
    __Pyx_RefNannySetupContext("process_byte_data", 0);

... etc
```

I thought that the two versions should produce almost identical code (annotations can be dismissed)